.gitattributes

```diff
@@ -1,3 +1,4 @@
+text=auto 
 # Ignore generated code in diffs.
-**/zz_generated.*.go linguist-generated=true
-/pkg/client/** linguist-generated=true
+**/zz_generated.*.go linguist-generated=true -diff
+/pkg/client/** linguist-generated=true -diff
```

Comment:
I am not a fan of making Git hide changes to these files. I occasionally do things like `git status` for the specific purpose of understanding what files got changed.